dev/raphael.svg.js

```diff
@@ -875,9 +875,6 @@ define(["./raphael.core"], function(R) {
             this.attr({"stroke-width": sw});
         }
 
-        //Reduce transform string
-        _.transform = this.matrix.toTransformString();
```

Comment:
thanks, God! this fixes issue with reset of some stacking transformations like transform('...S-1,1...S1.2') on a set of elements;